background.ts

```diff
@@ -218,6 +218,12 @@ Electron.app.whenReady().then(async() => {
       const message = err.message || err.toString();
 
       showErrorDialog(titlePart, message, true);
+      // if (err instanceof settings.SettingsError) {
+      //   // Even though we're passing fatal=true to showErrorDialog,
+      //   // this process still runs after the call to `Electron.app.quit()`,
+      //   // so stop processing this handler.
+      //   return;
+      // }
```

Comment:
Did you mean to have commented out code here? Is that needed?

Reply:
We are using version control, so should generally just delete code that isn't needed anymore.

If parts of the comments need to stay, then move them out of the conditional and make sure they are understandable on their own.

Reply:
Yes, it is needed. If we switch from `Electron.app.quit()` to `Electron.app.exit()` we don't but that change will need a lot of testing, and should be a separate issue.